packages/apollo-server-cloud-function/README.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Google Cloud Functions
+description: Setting up Apollo Server with Google Cloud Functions
+---
+
+[![npm version](https://badge.fury.io/js/apollo-server-cloud-function.svg)](https://badge.fury.io/js/apollo-server-cloud-function) [![Build Status](https://circleci.com/gh/apollographql/apollo-server.svg?style=svg)](https://circleci.com/gh/apollographql/apollo-server) [![Coverage Status](https://coveralls.io/repos/github/apollographql/apollo-server/badge.svg?branch=master)](https://coveralls.io/github/apollographql/apollo-server?branch=master) [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](https://www.apollographql.com/#slack)
+
+This is the Google Cloud Function integration of GraphQL Server. Apollo Server is a community-maintained open-source GraphQL server that works with many Node.js HTTP server frameworks. [Read the docs](https://www.apollographql.com/docs/apollo-server/v2). [Read the CHANGELOG](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md).
+
+```sh
+npm install apollo-server-cloud-function graphql
+```
+
+## Deploying with Google Cloud Function
+
+#### 1. Write the API handlers
+
+In a file named `graphql.js`, place the following code:
```

Comment:
I noticed it's actually necessary for the js file to be named index.js since the configuration only allows setting the directory of the function and not the file itself.